### ISSUE TITLE
kaltura- Internal server error occured

### ISSUE THREAD

**Issue by infra-oodles** [kaltura/server#3765]
Hi All,
I am using kaltura- 11.2.0 on single all in one server.
i have installed it all properly, but when i login through kmc, a error message popout saying-  **"Error- Internal server error occurred"**

running sanity test- all tests paased except DWH cycle

i am using, mysql  Ver 14.14 Distrib 5.1.73, for redhat-linux-gnu (x86_64) using readline 5.1

# rpm -qa "*kaltura*" - 
kaltura-release-11.6.0-1.noarch
kaltura-libopencore-amr-0.1.2-2.x86_64
kaltura-ffmpeg-2.7.2-2.x86_64
kaltura-monit-5.13-1.x86_64
kaltura-kdp-v2.7.0-1.noarch
kaltura-kvpm-v1.0.6-1.noarch
kaltura-kdpwrapper-v11.0-1.noarch
kaltura-kmc-v5.39.11-3.noarch
kaltura-libfaac-1.26-1.x86_64
kaltura-nginx-1.8.0-9.x86_64
kaltura-dwh-9.5.0-2.noarch
kaltura-x264-0.140-2.20140104.x86_64
kaltura-libass-0.9.11-2.x86_64
kaltura-a52dec-0.7.4-11.x86_64
kaltura-ffmpeg-aux-2.1.3-1.x86_64
kaltura-postinst-1.0.30-2.noarch
kaltura-base-11.2.0-12.noarch
kaltura-krecord-1.0.0-1.noarch
kaltura-sphinx-2.2.1-17.x86_64
kaltura-kdp3wrapper-v37.0-1.noarch
kaltura-flexwrapper-v1.2-1.noarch
kaltura-kupload-1.0.0-1.noarch
kaltura-kcw-1.0.0-6.noarch
kaltura-html5-studio-v2.0.1-1.noarch
kaltura-widgets-1.0.0-8.noarch
kaltura-librtmp-2.3-2.x86_64
kaltura-front-11.2.0-1.noarch
kaltura-mencoder-3.4.6-9.x86_64
kaltura-pentaho-4.2.1-2.x86_64
kaltura-server-11.2.0-1.noarch
kaltura-lame-3.99.5-3.x86_64
kaltura-sshpass-1.05-1.x86_64
kaltura-html5lib-v2.37.1-2.noarch
kaltura-clipapp-1.3-2.noarch
kaltura-kclip-v1.1.2.1-2.noarch
kaltura-kdp3-v3.9.9-2.noarch
kaltura-libmemcached-1.0.16-2.x86_64
kaltura-segmenter-1.0-2.x86_64
kaltura-batch-11.2.0-1.noarch

In logs i got-
2016-01-27 22:09:24 [1592080921] [addPermissionsAndItems.php] [addItemToPermissions] ALERT: exception 'Exception' with message 'ERROR - Permission name [BATCH_BASE] for partner [0] not found in database - skipping!' in /opt/kaltura/app/infra/log/KalturaLog.php:65
Stack trace:
#0 /opt/kaltura/app/alpha/scripts/utils/permissions/addPermissionsAndItems.php(280): KalturaLog::alert('ERROR - Permiss...')
--
  thrown in /opt/kaltura/app/batch/client/KalturaClientBase.php on line 894
PHP Fatal error:  Uncaught exception 'KalturaException' with message 'Internal database error' in /opt/kaltura/app/batch/client/KalturaClientBase.php:894
Stack trace:
#0 /opt/kaltura/app/batch/client/KalturaClient.php(1294): KalturaClientBase->throwExceptionIfError(Array)
--
  thrown in /opt/kaltura/app/batch/client/KalturaClientBase.php on line 894
PHP Fatal error:  Uncaught exception 'KalturaException' with message 'Internal database error' in /opt/kaltura/app/batch/client/KalturaClientBase.php:894
Stack trace:
#0 /opt/kaltura/app/batch/client/KalturaClient.php(1132): KalturaClientBase->throwExceptionIfError(Array)
-- # tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color "ERR:\|PHP\|trace\|CRIT\|\[error\]"
  thrown in /opt/kaltura/app/batch/client/KalturaClientBase.php on line 894
PHP Fatal error:  Uncaught exception 'KalturaException' with message 'Internal database error' in /opt/kaltura/app/batch/client/KalturaClientBase.php:894
Stack trace:
#0 /opt/kaltura/app/batch/client/KalturaClient.php(634): KalturaClientBase->throwExceptionIfError(Array)
--
  thrown in /opt/kaltura/app/batch/client/KalturaClientBase.php on line 401
PHP Fatal error:  Uncaught exception 'KalturaClientException' with message 'failed to unserialize server result
' in /opt/kaltura/app/batch/client/KalturaClientBase.php:401
Stack trace:
#0 /opt/kaltura/app/batch/client/KalturaClient.php(1225): KalturaClientBase->doQueue()
--
  thrown in /opt/kaltura/app/batch/client/KalturaClientBase.php on line 894
PHP Fatal error:  Uncaught exception 'KalturaException' with message 'Internal database error' in /opt/kaltura/app/batch/client/KalturaClientBase.php:894
Stack trace:
#0 /opt/kaltura/app/batch/client/KalturaClient.php(1117): KalturaClientBase->throwExceptionIfError(Array)
--
  thrown in /opt/kaltura/app/batch/client/KalturaClientBase.php on line 894
PHP Fatal error:  Uncaught exception 'KalturaException' with message 'Internal database error' in /opt/kaltura/app/batch/client/KalturaClientBase.php:894
Stack trace:
#0 /opt/kaltura/app/batch/client/KalturaClient.php(4124): KalturaClientBase->throwExceptionIfError(Array)

**Comment by kaltura-hooks** [comment]
Hi @infra-oodles,

Thank for you reporting an issue and helping improve Kaltura!

To get the fastest response time, and help the maintainers review and test your reported issues or suggestions, please ensure that your issue includes the following (please comment with more info if you have not included all this info in your original issue):
 - Is the issue you're experiencing consistent and across platforms? or does it only happens on certain conditions?
    please provide as much details as possible.
 - Which Kaltura deployment you're using: Kaltura SaaS, or self-hosted?
	If self hosted, are you using the RPM or deb install?
 - Packages installed.
	When using RPM, paste the output for:
```
	# rpm -qa "kaltura*"
```	
	For deb based systems:
```	
	# dpkg -l "kaltura-*"
```
 - If running a self hosted ENV - provide the MySQL server version used
 - If running a self hosted ENV - is this a single all in 1 server or a cluster?
 - If running a self hosted ENV, while making the problematic request, run:
```
	# tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color "ERR:\|PHP\|trace\|CRIT\|\[error\]"
```
   and paste the output.
 - When relevant, provide any screenshots or screen recordings showing the issue you're experiencing.

For general troubleshooting see:
https://github.com/kaltura/platform-install-packages/blob/Jupiter-10.13.0/doc/kaltura-packages-faq.md#troubleshooting-help

If you only have a general question rather than a bug report, please close this issue and post at:
http://forum.kaltura.org

Thank you in advance,

**Comment by infra-oodles** [comment]
Hi guys, I have solved this issue, firstly i tried each and every troubleshooting steps, including a fresh installation many times but that didn't helped. I also tried by installing a new mysql server with fresh installation but that also didn't helped. Then i noticed the mysql user's (etl, kaltura, root) permissions, password, hosts. I dropped etl & kaltura users and then did a fresh installation and that solved my issue.
